components/more_direct_channels/more_direct_channels.jsx

```diff
@@ -175,9 +186,13 @@ export default class MoreDirectChannels extends React.Component {
     handlePageChange(page, prevPage) {
         if (page > prevPage) {
             if (this.listType === 'any') {
```

Comment:
This has similar signature as above (LOC 80-88). Might be better to be on it's own function.